src/cli/commands/migrate.ts

```diff
@@ -32,7 +42,23 @@ type Options = { run: boolean; revert: boolean; show: boolean; tenant_codes: str
  * @param tenant_codes
  */
 export async function migrate({ run, revert, show, tenant_codes }: Options): Promise<void> {
-	const { configModule } = getConfigFile(process.cwd(), `medusa-config`) as { configModule: ConfigModule };
+	//const { configModule } = getConfigFile(process.cwd(), `medusa-config`) as { configModule: ConfigModule };
+	const configuration = getConfigFile(process.cwd(), `medusa-config`) as {
+		configModule: ConfigModule;
+		configFilePath: string;
+	};
+	const resolveConfigProperties = async (obj): Promise<ConfigModule> => {
```

Comment:
todo: from the comment above you will be able to use the util instead of duplicating the code

Reply:
done